src/styles/components/_subscribe.scss

```diff
@@ -0,0 +1,79 @@
+@import '../settings/colors';
+@import '../tools/breakpoints';
+
+.w-subscribe {
+  display: block;
+  margin-top: 32px;
+  text-align: center;
```

Comment:
It seems like you don't want this here. Instead you probably want to move this to the div that's the first child of w-subscribe and contains the h1. That seems to be the only place where you actually want centered text.